### DISCUSSION TITLE
Welcome to Discussions for the PC's Infinite Ammo mod.

### DISCUSSION THREAD

**Discussion by ProfessorConfundus** [ProfessorConfundus/PCs-Infinite-Ammo#1] [Announcements]
## Hi!
  Discussions are used here for: 
  * Asking questions not answered in [the FAQ](https://mods.factorio.com/mod/PCs-Infinite-Ammo/faq) at the mod page.
  * Sharing possible ideas and improvements for the mod.

  If you have any problems with the mod, like bug reports, incompatibilities or inefficiencies with the mod, please open an **Issue** instead of a discussion, cheers.

**Comment by karnoss** [comment]
Hello! i was using this mod to have fun with they are biter (make spawn a lot of biter)
everything working fine until i update the mod today and got that error message
![image](https://user-images.githubusercontent.com/99127501/152683035-57f23fd3-42db-4953-89eb-eb2627cc37d0.png)

**Reply by ProfessorConfundus** [reply]
Hi there! 

What other mods are you playing the game with?

**Reply by karnoss** [reply]
i was doing test to see what mods don't work with
and i found the one  -> Bob's Warfare

**Reply by karnoss** [reply]
the strange thing, they was no issue before the update.

**Reply by ProfessorConfundus** [reply]
I should've known Bob's mods would've caused issues, heh! 
Just one last question; does the Scatter Bomb Shell have an icon? The issue seems to be related to a missing icon. I would test and see myself right now if I could, but it's 3am where I am in the world right now, so that also means the update might take a few hours to release.
Sorry for any inconvenience!

**Reply by karnoss** [reply]
no problem!
yes, that was different type of ammo
i make ew world to see the ammo type ad here he is! 
![image](https://user-images.githubusercontent.com/99127501/152691104-a7d06380-fe21-4d90-a13e-4c419b9d06ce.png)

**Reply by ProfessorConfundus** [reply]
@karnoss The issue has been fixed in an update to the mod. Thank you so much for reporting the bug!
Also, PS: If you ever want to submit a bug report again for this mod, please use an issue, not discussions. Discussions here is used for ideas, suggestions and general conversation.

**Reply by ProfessorConfundus** [reply]
I also have no clue in the slightest how the issue only appeared after updating to 0.2.0. I made absolutely naught changes to the code that handles icon generation in 0.2.0.

**Reply by karnoss** [reply]
thank you, and sry about that, first time using github, and i was pretty lost to how to use it

**Reply by ProfessorConfundus** [reply]
No problem. Also yeah, GitHub can be intimidating at first; I still barely know how to edit a pull request, and I've been on here for a while!
